src/games/RomSettings.cpp

```diff
@@ -17,6 +17,8 @@
  */
 #include "RomSettings.hpp"
 
+RomSettings::RomSettings() : m_mode(0) {}
```

Comment:
The m_mode variable doesn't seem to be used. Will it be used?

Unless it's needed here, we should push it to be individual ROMSettings subclasses for consistent with m_score, m_reward, etc.

Reply:
Fixed.